aws/tailers/stack_event.go

```diff
@@ -9,22 +10,51 @@ import (
 	"text/tabwriter"
 
 	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go/service/configservice"
 	"github.com/fatih/color"
 	"github.com/wallix/awless/aws/services"
 )
 
+const (
+	FilterStackEventLogicalID    = "id"
+	FilterStackEventTimestamp    = "ts"
+	FilterStackEventStatus       = "status"
+	FilterStackEventStatusReason = "reason"
+	FilterStackEventType         = "type"
```

Comment:
Let just call them:
```go
EventLogicalID     = "id"
EventTimestamp    = "ts"
EventStatus       = "status"
EventReason      = "reason"
EventType        = "type"
```

Reply:
Since all "tailors" are located in the same package, it could cause some confusion when we add more tailors, isn't it?

Reply:
True. Then prefix with Stack: `StackEvent...`. Also people that will add tailers will worry about that. Let's not worry about "possible's future confusion" but more about "today's confusion"